src/main/java/org/skillset/mystays/AccountService.java

```diff
@@ -18,14 +19,24 @@ public List<Account> get() {
     }
 
     @Transactional
-    public void insert(Account account) {
-        AccountEntity ac = new AccountEntity();
-        ac.email = account.getEmail();
-        ac.firstName = account.getFirstName();
-        ac.lastName = account.getLastName();
-        ac.password = account.getPassword();
-        ac.phoneNumber = account.getPhoneNumber();
-        ac.persist();
+    public Boolean insert(Account account) {
+
+        try {
+
+            AccountEntity ac = new AccountEntity();
+            ac.email = account.getEmail();
+            ac.firstName = account.getFirstName();
+            ac.lastName = account.getLastName();
+            ac.password = account.getPassword();
+            ac.phoneNumber = account.getPhoneNumber();
+            ac.persist();
+            System.out.println(status);
+            return status;
+        }
+        catch(Exception e) {
+            return !status;
+        }
```

Comment:
Please give a Sysout saying "Account Creation failed"
you can directly return false